manifests/manifest-prod-web-services.yml

```diff
@@ -0,0 +1,18 @@
+---
+applications:
+  - name: fecfile-web-services
+    instances: 2
+    stack: cflinuxfs4
+    buildpacks:
+      - python_buildpack
+    memory: 1G
+    no-route: true
+    health-check-type: process
+    command: bin/run-celery-worker.sh
+    services:
+      - fecfile-api-rds
+      - fecfile-api-s3
+      - fecfile-api-redis
```

Comment:
Can we add the creds service as well? The web service needs the FEC_FILING_API_KEY and we're currently storing it in both the web service app variables and the creds service. It'd be this addition:

      - fecfile-api-creds-prod

Similarly, dev and stage are fecfile-api-creds-dev and fecfile-api-creds-stage for those manifests.

Reply:
@mjtravers makes sense to me! Would you like the work to tell Celery to look at the creds service instead of its own env var to be its own ticket? We'll have to wait until the changes are merged/deployed in all the spaces, so it'll probably be a new sprint.

Reply:
Yes, let's get the manifests updated in this ticket and we can rearrange the application variables in a later ticket after the updated manifests have worked their way through the 3 spaces.

Reply:
@mjtravers manifest changes made, follow-up ticket here: https://github.com/fecgov/fecfile-web-api/issues/644